what-if.md

```diff
@@ -0,0 +1,15 @@
+# What if I need to...?
+
+This document describes possible operations that developers may need to perform.
+
+## Integrating a new city
```

Comment:
Also add parts for:

- Adding a new vendor
  - deliver new MQTT credentials to them securely, username and password per environment
  - for RabbitMQ give them read/write access to their own subtree in MQTT topic space for staging and possibly only read access to production. So one vendor should only see their own data.
  - monitor their behaviour in staging first
- A new vehicle gets APC devices
  - nothing needs to be done but that is good to write down
- If journey-matcher load is too high, separate one journey-matcher per municipality by copying the manifest structure of city-specific manifests